### PR TITLE
Add fbox.to presence

### LINKED ISSUE #3985: FBOX | www3.fbox.to
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** <Rushifā#6129>

**What's the name of the service?** <www3.fbox.to>

**What should the Presence display?** <like gogoanime/ animepahe it should show what i'm watching (for movie--> Movie title) (Fore series--> Season & Episode) , searching/ browsing, time left, watch now button, if possible maybe the poster of the movie>

**If possible, please provide a logo for the service (512x512 minimum)**

### PR DESCRIPTION
**Pull request by vahya**
Closes PreMiD/Presences#3985

Screenshot 2021-07-05 174155
Screenshot 2021-07-05 174310
Screenshot 2021-07-05 174449
Screenshot 2021-07-05 174518

### CODE REVIEW

File: websites/F/fbox.to/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "fbox.to",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "videos",
+  "logo": "https://i.imgur.com/DIx68rI.png",
```
Comment (ririxidev) [inline websites/F/fbox.to/dist/metadata.json:9]: that's low quality logo

File: websites/F/fbox.to/presence.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    startTimestamp: browsingStamp,
+    largeImageKey: "fbox_logo"
+  },
+    { pathname } = document.location;
+
+  if (pathname === "/") presenceData.details = "Exploring Fbox.to";
```
Comment (ririxidev) [inline websites/F/fbox.to/presence.ts:26]:
```suggestion
  if (pathname === "/") presenceData.details = "Browsing";
```

**Review by rxri — CHANGES_REQUESTED** [review]
Support every subpage

File: websites/F/fbox.to/dist/metadata.json (hunk 1 above)
Comment (vahya) [reply websites/F/fbox.to/dist/metadata.json]: fbox_logo
Changed it to this, it should be okay now.

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**